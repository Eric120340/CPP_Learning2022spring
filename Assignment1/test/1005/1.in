3 3 7433921
586369712 797237589 612408761 
989352026 417461619 340839794 
885518784 697655629 390901806 
