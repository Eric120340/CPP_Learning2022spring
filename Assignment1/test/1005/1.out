528367646 753071147 1571327 
596890610 365777060 286658448 
427802720 941443693 971395662 
